lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -331,7 +332,7 @@ public void DeliverTaskMessage(byte[] message)
             }
         }
 
-        public void HandleContextMessaage(byte[] mesage)
+        public void HandleContextMessage(byte[] mesage)
```

Comment:
@dongjoon-hyun I've sent out an email regarding this to @dev a few days ago. Provided this is a class that shouldn't be explicitly used by our users, and nobody objected, I didn't bother to deprecate it, but perhaps it's safer this way. Thanks!